src/glide/browser/base/content/browser.mts

```diff
@@ -1665,6 +1665,12 @@ class GlideOptions implements GlideO {
     this.#hint_size = value;
     GlideBrowser.set_css_property("--glide-hint-font-size", value);
   }
+
+  // Ideally this would default to glide.api.hints.label_prefix_free,
+  // but we haven't made the api object yet! This may be an indication that
```

Comment:
One option is to use a getter/setter like the above option, so that it's lazily initialised.

e.g.
```typescript
#hint_label_generator: glide.Options['hint_label_generator'] | null = null;

get hint_label_generator() {
  return this.#hint_label_generator ?? GlideBrowser.api.hints.label_prefix_free;
}
set hint_size(value: glide.Options['hint_label_generator']) {
  this.#hint_label_generator = value;
}
```

But defining the default label generator in `hinting.mts` and then just referencing it here would also be a fine approach.

Reply:
I think that's much nicer than having to do the defaulting at the point of use. For now there's only one such point of use, but it's possible we could grow more. And even without that, having the default defined here alongside the Options object is a much more sensible place.